src/auth-service/utils/role-permissions.util.js

```
@@ -104,7 +104,1126 @@
+          roleCreationResults.push(result);
+        }
+      } catch (error) {
+        console.error(
+          `❌ Failed to create role ${roleData.role_name}: ${error.message}`
+        );
+        // Continue with other roles even if one fails
+        continue;
+      }
+    }
+
+    console.log(
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)